app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import mongoose from '../services/mongoose.js';
+
+const name = 'inspiration'
```

Comment:
It's a convention to name string constants in all `UPPERCASE` like `const NAME = 'inspiration'`